src/core/pal/feature.h

```diff
@@ -265,16 +265,16 @@ namespace pal
 
       /**
        * Returns the label position for a curved label at a specific offset along a path.
-       * \param path_positions line path to place label on
-       * \param path_distances array of distances to each segment on path
+       * \param mapShape line path to place label on
```

Comment:
@nyalldawson , mapShape or pathShape?

Thanks for the extra cleanup.

Reply:
I've gone with "mapShape" to match all the other methods in this class

Reply:
You undoubtedly know best :)